docs_src/security/Ch-CORS-Settings.md

```diff
@@ -0,0 +1,29 @@
+# CORS enabling
+
+!!! edgey "EdgeX 2.1"
+    New for EdgeX 2.1 is the ability to enable CORS access to EdgeX microservices through configuration. 
+
+The EdgeX microservices provide REST APIs and those services might be called from a GUI through a browser. Most browsers prevent service calls from a different origin. Thus, EdgeX supports Cross-Origin Resource Sharing (CORS) since Jakarta release (v2.1), and this feature can be controlled by the configurations. The default behavior of CORS is disabled. Here is a good reference to understand [CORS](https://www.html5rocks.com/en/tutorials/cors//).
```

Comment:
```suggestion
The EdgeX microservices provide REST APIs and those services might be called from a GUI through a browser.  Browsers prevent service calls from a different origin, making it impossible to host a management GUI on one domain that manages an EdgeX device on a different domain. Thus, EdgeX supports Cross-Origin Resource Sharing (CORS) since Jakarta release (v2.1), and this feature can be controlled by the configurations. The default behavior of CORS is disabled. Here is a good reference to understand [CORS](https://www.html5rocks.com/en/tutorials/cors/).
```

Reply:
@bnevis-i the page of https://www.html5rocks.com/en/tutorials/cors// and https://www.html5rocks.com/en/tutorials/cors/ are different

Reply:
@cloudxxx8 The final "/" on the URL is redundant.  I tried both, and both are redirecting to https://web.dev/cross-origin-resource-sharing/ for me.